ui/webpack.config.js

```diff
@@ -76,6 +77,7 @@ module.exports = (env) => {
         output: {
             path: path.join(__dirname, '/dist'),
             filename: 'bundle.js',
+            publicPath: '', // For font preloader plugin, otherwise it generates a broken prefix for font file names
```

Comment:
Jonkun bugin seurauksena jokaisen linkin href-attribuutin alkuun generoitui merkkijono "auto" (joka on publicPathin default-arvo, mutta toimii samaan tapaan kuin esim. CSS:n termi "auto", eli ei suoraan tarkoita että ka. arvo olisi kirjaimellisesti tuo merkkijono.) Joku oli korjannut githubin issuesissa näin, joten samalla korjauksella mennään.